labgraph/graphs/group.py

```diff
@@ -170,6 +170,12 @@ def _get_streams(self) -> Dict[str, Stream]:
                     if topic is descendant_topic:
                         topic_paths.append(topic_path)
                         break
+            
+            # print("TOPIC PATHS", topic_paths)
+            
+            if len(topic_paths) == 0:
```

Comment:
This is used to debug because there are empty `topic_paths` for some reason, which throws an error but doesn't cause problems